detox/e2e/support/server_api/common.js

```diff
@@ -1,8 +1,30 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
+import FormData from 'form-data';
+import fs from 'fs';
+
+import client from './client';
+
 export const getResponseFromError = (err) => {
     const {response: {data, status}} = err;
 
     return {error: data, status};
 };
+
+export const apiUploadFile = async (name, absFilePath, requestOptions = {}) => {
+    const formData = new FormData();
+    formData.append(name, fs.createReadStream(absFilePath));
```

Comment:
you should check that the absFilePath actually exists before reading the file contents. I know you  are checking for file existence before calling this method, but it should be this method that checks for that.

Reply:
Done.